docs/index.html

```diff
@@ -1556,23 +1558,44 @@ <h6 id="name-semantics-9">
 </section>
 </div>
 <section id="section-2.6.2.4">
-            <h5 id="name-array-index-2">
-<a href="#section-2.6.2.4" class="section-number selfRef">2.6.2.4. </a><a href="#name-array-index-2" class="section-name selfRef">Array Index</a>
+            <h5 id="name-array-access-2">
+<a href="#section-2.6.2.4" class="section-number selfRef">2.6.2.4. </a><a href="#name-array-access-2" class="section-name selfRef">Array Access</a>
             </h5>
 <section id="section-2.6.2.4.1">
               <h6 id="name-syntax-10">
 <a href="#name-syntax-10" class="section-name selfRef">Syntax</a>
               </h6>
-<p id="section-2.6.2.4.1-1">An array index is a union element consisting of an integer (in base 10).<a href="#section-2.6.2.4.1-1" class="pilcrow">¶</a></p>
+<p id="section-2.6.2.4.1-1">Array access is a union element which matches any node which is an array and which selects zero or more elements of the array.
+                        Array access takes the form of an index, which selects at most one element, or a slice, which selects zero or more elements.<a href="#section-2.6.2.4.1-1" class="pilcrow">¶</a></p>
```

Comment:
The expression like `$[0, 1]` will not be included in the spec?

Reply:
Yes it will. According to the syntax definition `[0, 1]` is a union of two array accesses.

Reply:
Oh, I forgot that index access is a type of union element, sorry.

Reply:
No problem. If others make the same mistake, we may need to think about how to address that in the document.

(I am trying to avoid adding numerous examples, which might be one fix, as that can weaken a standards document because less attention is paid to the normative text, by both readers and authors.)